test/test_mcmc.py

```diff
@@ -50,8 +50,8 @@ def model(labels):
     hmc_state = init_kernel(init_params,
                             trajectory_length=10,
                             num_warmup_steps=warmup_steps)
-    hmc_states = fori_append(sample_kernel, hmc_state, num_samples,
-                             transform=lambda x: transform_fn(x.z))
+    hmc_states = fori_collect(num_samples, sample_kernel, hmc_state,
```

Comment:
I found this to be the case with the version of `fori_append` that I was working with as well and it seemed like this test is particularly impacted due to the large number of samples that we draw. To check this hypothesis, maybe we can increase the number of samples in other tests and see how the comparison holds up.

Reply:
Yup, I think that some tests with 20000 samples will be a good benchmark for us. I'll make another pass with 20000 samples after investigating the slowness of logistic regression model.